tests/exception_on_exception.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from logging import getLogger
+
+from logger_tt import setup_logging
+
+__author__ = "ZeroRin"
+
+setup_logging(analyze_raise_statement=True)
+logger = getLogger(__name__)
+
+def foo():
+    try:
+        a=1
+        raise RuntimeError(a)
+    except:
+        try:
+            b=2
+            raise RuntimeError(b)
+        except:
+            c=3
+            raise RuntimeError(c)
+
+if __name__ == '__main__':
+    match sys.argv[-1]:
```

Comment:
Using the `match` syntax is good only if this code is run from py 3.10+.
This test will fail for any older python version.
Since this library supports python 3.6+, please just use the `if-else` syntax.